app/services/actions/foreman/report/import.rb

```diff
@@ -0,0 +1,37 @@
+module Actions
+  module Foreman
+    module Report
+      class Import < Actions::EntryAction
+        def resource_locks
+          :import_reports
+        end
+
+        def plan(params, report_class, detected_proxy_id)
+          plan_self :params => params, :report_class => report_class.to_s, :detected_proxy_id => detected_proxy_id
+        end
+
+        def run
+          report_class = input[:report_class].constantize
+          report = report_class.import(input[:params][:report], SmartProxy.find_by_id(input[:detected_proxy_id]))
```

Comment:
This was changed a while ago https://github.com/theforeman/puppet-foreman/pull/466 but the new version of module will be released with 1.14 I guess. For backwards compatibility, I'll look to :config_reports as fall back.